src/selectbox_abstract.coffee

```diff
@@ -108,6 +108,8 @@ class SelectBoxAbstract
         # Replicate changes from the original select box to the fake one
         @orig.on 'reform.sync change DOMSubtreeModified', @refresh
         
+        @body.on 'reform.close', => @close()
```

Comment:
I have a situation on the preview. If you drag a panel every open div on the panel should be closed else it just stays on the screen... We kinda do the same with `reform.open`.